src/Game.js

```diff
@@ -1,9 +1,28 @@
+const Card = require('./Card');
 const data = require('./data');
+const Deck = require('./Deck');
+const Round = require('./Round');
 const prototypeQuestions = data.prototypeData;
 const util = require('./util');
 
 class Game {
-  constructor() {}
+  constructor() {
+    // this.deck;
+    this.turns;
+    this.currentRound = {};
+  }
+
+  start() {
+    const playingCards = prototypeQuestions.map(question => {
```

Comment:
This was really tricky, and I am glad I was able to push through and work on figuring it out.